### PR TITLE
1559: Minor touchup work.

### PR DESCRIPTION
**Pull request by MicahZoltu**

### CODE REVIEW

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - eip-1559.md requires approval from one of (vbuterin, econoar, afdudley, mslipper, i-norden, abdelhamidbakhta)
 @vbuterin @econoar @afdudley @mslipper @i-norden @abdelhamidbakhta

File: EIPS/eip-1559.md
Comment (MicahZoltu) [reply EIPS/eip-1559.md:18]: @axic within document consistency is higher priority than cross-dacument consistency, which is why I changed it here.  I support fixing the other two as well, and bringing consistency across them.

@timbeiko We can, though this change is just bringing stylistic consistency to the docu ent so I can't imagine someone wanting to block this PR over it (I interpret Axic's comment to just be that we should also make this change elsewhere).  Also, I'm away from my computer for a while so someone else will need to submit the split out PRs if you want it super soon.

Comment (timbeiko) [reply EIPS/eip-1559.md:18]: @axic are you OK for us to merge this in and we can update 2718+2930 in a follow up PR?

Comment (axic) [reply EIPS/eip-1559.md:18]: It was only a friendly remark as I remember it was discussed on one of the channels.

Comment (timbeiko) [reply EIPS/eip-1559.md:18]: Got it. Sorry about being pushy! Just want to make sure we minimize the number of blockers 😅

**Review by abdelhamidbakhta — APPROVED** [review]
Looks good to me

**Pull request merged**
